doc/releasenotes/15_0_0_summary.md

```diff
@@ -75,6 +75,9 @@ The following VTTablet flags were deprecated in 7.0. They have now been deleted
 - --enable_semi_sync is now deprecated. It will be removed in v16. Instead, set the correct durability policy using `SetKeyspaceDurabilityPolicy`
 - --queryserver-config-pool-prefill-parallelism, --queryserver-config-stream-pool-prefill-parallelism and --queryserver-config-transaction-prefill-parallelism have all been deprecated. They will be removed in v16.
 
+#### vtbackup flag deprecations
+- --backup_storage_hook is deprecated; Consider using ExternalCompressorCmd/ExternalDecompressorCmd instead.
```

Comment:
Should be changed to match flag deprecation message. Note `Consider` should be `consider` (lowercase).

Reply:
fixed ..